phpBB/phpbb/update/get_updates.php

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\update;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use phpbb\filesystem\exception\filesystem_exception;
+use phpbb\filesystem\filesystem_interface;
+use SodiumException;
+use ZipArchive;
```

Comment:
composer should include php-zip as requirement
There is a `functions_compress.php` in includes folder too that can be used if the server don't have zip installed (or just dont allow updates)

Reply:
i doubt that would actually work if this is not working.

Reply:
It wouldn't, I did however add the composer requirement now.